config/initializers/markdown.rb

```diff
@@ -0,0 +1,7 @@
+MarkdownRails.handle :md, :markdown do
+  ApplicationMarkdown.new
+end
+
+MarkdownRails.handle :markerb do
```

Comment:
What about using `.md.erb` for the file extension?

Reply:
The convention in Rails is that the rightmost extension is the template engine and the middle extension is the response format. So it should be `<basename>.html.<something>`, having more extensions are just ignored e.g. `<basename>.html.md.erb` wouldn't work (as md isn't a response format).

Tbh, I just used `markerb` as it was in the markdown-rails gem docs, but could change it to something `mderb` if it matters.